crates/net/network/src/session/mod.rs

```diff
@@ -656,7 +672,14 @@ async fn start_pending_outbound_session(
     bandwidth_meter: BandwidthMeter,
 ) {
     let stream = match TcpStream::connect(remote_addr).await {
-        Ok(stream) => MeteredStream::new_with_meter(stream, bandwidth_meter),
+        Ok(stream) => {
+            // Instantiating a new [`BandwidthMeterMetrics`] struct here to get per-stream metrics.
+            // NOTE: This means the metric values will be <= the in/outbound values in the
+            // [`BandwidthMeter`], as it is shared by all streams.
+            let bandwidth_metrics =
+                BandwidthMeterMetrics::new(&format!("bandwidth_session_{session_id}"));
```

Comment:
interesting - @mattsse wdyt?

Reply:
need to use metric labelling as introduced by #892 instead